client/routes/namespace/workflow-list.vue

```diff
@@ -130,12 +130,15 @@ export default pagedGrid({
       statuses: [
         { value: 'OPEN', label: 'Open' },
```

Comment:
I would rename this to `RUNNING` for consistency with `WORKFLOW_EXECUTION_STATUS_RUNNING` (if it makes sense).

Reply:
in general I agree, in this case we should also update the API and remove explicit ListOpenWorkflowExecutions command (as well as ListClosedWorkflowExecutions ). Instead unify into a single endpoint ListWorkflowExecutions with a status param